src/otx/algorithms/visual_prompting/adapters/pytorch_lightning/models/backbones/tiny_vit.py

```diff
@@ -0,0 +1,744 @@
+"""TinyViT for MobileSAM."""
+
+# Copyright (c) 2022 Microsoft
```

Comment:
@yunchu I left the license that MobileSAM uses, as you suggested before for [visual prompting task](https://github.com/openvinotoolkit/training_extensions/blob/develop/src/otx/algorithms/visual_prompting/adapters/pytorch_lightning/models/visual_prompters/segment_anything.py#L7-L9), could you check if it is correct?